WalletWasabi/Stores/IndexStore.cs

```diff
@@ -167,6 +170,11 @@ private void MigrateToSqliteNoLock()
 
 			Logger.LogInfo($"Migration of {i} filters to SQLite was finished in {stopwatch.Elapsed} seconds.");
 		}
+		catch (OperationCanceledException ex)
```

Comment:
We don't use the `ex`
```suggestion
		catch (OperationCanceledException)
```

Reply:
We could log it instead of removing.

Reply:
The user quits, it makes no sense to log.